### ISSUE TITLE
Not enough time for target transition sequence 

### ISSUE THREAD

**Issue by cqu20160901** [nl8590687/ASRT_SpeechRecognition#32]
您好，按着你的数据读入方式， 把模型改成tensoflow版的出现以下问题：
InvalidArgumentError (see above for traceback): Not enough time for target transition sequence (required: 64, available: 39)0You can turn this error into a warning by using the flag ignore_longer_outputs_than_inputs
	 [[Node: loss/CTCLoss = CTCLoss[ctc_merge_repeated=true, ignore_longer_outputs_than_inputs=false, preprocess_collapse_repeated=false, _device="/job:localhost/replica:0/task:0/cpu:0"](output_logits/_31, _arg_input_label/indices_0_0, _arg_input_label/values_0_2, _arg_seq_len_0_4)]]

我用CTCLoss中ignore_longer_outputs_than_inputs=True 代码可以跑通，但是训练损失会出现 inf .

纠结了很久，不知道如何解决？

**Comment by nl8590687** [comment]
你看报错信息，把39调成64试试

这个是TensorFlow的CTC的问题，您可能没有理解CTC的原理，建议去查阅一下相关资料。
